Lib/ctypes/test/test_byteswap.py

```diff
@@ -311,5 +311,68 @@ class S(Structure):
         s2 = struct.pack(fmt, 0x12, 0x1234, 0x12345678, 3.14)
         self.assertEqual(bin(s1), bin(s2))
 
+    def test_union_fields(self):
```

Comment:
IMO this test would be better named something like `test_union_fields_unsupported_byte_order`.